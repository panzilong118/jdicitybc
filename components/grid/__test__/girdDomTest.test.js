/**
 * Created by huangxiao3 on 2018/3/20.
 */
import * as React from "react";
import renderer from 'react-test-renderer';
import { Col, Row } from '../index';
describe('<Input/>', () => {
    it('demo test Snapshot', () => {
        const componentGrid = renderer.create(
            <Row gutter={16}>
                <Col span={8}>
                    123
                </Col>
                <Col span={8}>
                    456
                </Col>
                <Col span={8}>
                    789
                </Col>
            </Row>
        );
        let snapsGrid = componentGrid.toJSON();
        expect(snapsGrid).toMatchSnapshot();
    });
});