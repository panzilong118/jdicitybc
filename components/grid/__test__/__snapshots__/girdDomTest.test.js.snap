// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input/> demo test Snapshot 1`] = `
<div
  className="jc-row"
  style={
    Object {
      "marginLeft": -8,
      "marginRight": -8,
    }
  }
>
  <div
    className="jc-col-8"
    style={
      Object {
        "paddingLeft": 8,
        "paddingRight": 8,
      }
    }
  >
    123
  </div>
  <div
    className="jc-col-8"
    style={
      Object {
        "paddingLeft": 8,
        "paddingRight": 8,
      }
    }
  >
    456
  </div>
  <div
    className="jc-col-8"
    style={
      Object {
        "paddingLeft": 8,
        "paddingRight": 8,
      }
    }
  >
    789
  </div>
</div>
`;
